lavorare-insieme/dove-quando-e-come.md

```diff
@@ -0,0 +1,83 @@
+# Dove, quando e come
+
+La nostra settimana lavorativa si costituisce di 4 giorni dedicati a lavori per clienti esterni (non
+necessariamente sempre con lo stesso cliente) e di 1 giorno,
+[il venerdì](https://github.com/nebulab/playbook/blob/master/crescita-personale/venerdi.md), dedicato
+solo all'apprendimento, ai progetti interni o al contributo open source.
+
+Le giornate lavorative sono di 8 ore piuttosto flessibili. Per legge non si possono comunque
+superare le 13 ore giornaliere e le 48 ore settimanali. L'orario adottato dalla maggior parte
+di noi è compreso tra le 9:30 e le 18:30 considerando un'ora circa per la pausa pranzo.
+L'organizzazione del lavoro deve essere comunque vista nell'ottica del lavoro in team.
+Quindi chi sceglie di lavorare in orari diversi deve essere attento a non impedire ad un
+altro di svolgere correttamente il proprio lavoro.
+
+Riguardo al luogo la flessibilità è ancora maggiore. Nebulab al momento ha due sedi operative, una a
+Pescara e una a Latina, ma si è liberi di lavorare ovunque si voglia (volendo anche da casa
+propria).
+
+Questa flessibilità in termini di orario e luogo è resa possibile dalla disciplina del nostro team e
+dall'uso continuo e attento di strumenti ad hoc.
+
+## Lavorare da remoto
+
+Come menzionato sopra, è possibile in qualunque momento (con l'opportuno preavviso) scegliere di
+lavorare da remoto per un periodo determinato o a tempo indefinito. In Nebulab crediamo fermamente
+nel fatto che tutti dovrebbero lavorare dove sono più produttivi e nel valore aggiunto di una cultura
```

Comment:
Questa frase la cambierei in `In Nebulab crediamo fermamente nella filosofia remote-first, in quanto non solo aumenta la produttività del singolo che si trova a lavorare in un ambiente a lui più congeniale, ma soprattutto instilla nel team l'abitudine alla comunicazione continua e alla trasparenza.` rimuovendo la entry relativa a `remote-first philosophy` nella lista che segue ` Principi fondamentali`. My two cent.

Reply:
@MinasMazar cambio la frase, ma penso di lasciare il principio nella sezione dopo. Voglio che sia chiaro che il lavoro da remoto e il lavoro in sede sono sullo stesso piano sia in quanto a "valore" portato dal lavoratore che in quanto a responsabilità.